packages/schema-blocks/src/functions/innerBlocksHelper.ts

```diff
@@ -33,4 +34,15 @@ function filterBlocksRecursively( blockInstance: BlockInstance, predicate: ( blo
 	return foundBlocks;
 }
 
-export { filterBlocksRecursively, getInnerblocksByName };
+/**
+ * Inserts a block to the inner block.
+ *
+ * @param {BlockInstance} block    The block to insert.
+ * @param {string}        clientId Id of the element to insert the block to.
+ * @param {number}        index    The location of the block.
+ */
+function insertBlockToInnerBlock( block: BlockInstance, clientId: string, index?: number ) {
```

Comment:
This should have a `void` return type.
```suggestion
function insertBlockToInnerBlock( block: BlockInstance, clientId: string, index?: number ): void {
```